src/bpftrace.cpp

```diff
@@ -994,7 +994,13 @@ int BPFtrace::poll_skboutput_events()
 
 void BPFtrace::poll_event_loss(Output &out)
 {
-  auto current_value = bytecode_.get_event_loss_counter(*this);
+  uint64_t current_value = 0;
+  uint64_t *current_values = bytecode_.get_event_loss_counter(*this);
+  for (int i = 0; i < max_cpu_id_; ++i) {
+    current_value += *current_values;
+    current_values++;
+  }
```

Comment:
It feels like the aggregation should be done in `bytecode_.get_event_loss_counter`, shouldn't it?